### PR TITLE
Programmes list

### PR DESCRIPTION
**Pull request by davidrapson**
For https://github.com/biglotteryfund/blf-alpha/issues/275

localhost-3000-funding-programmes-min 1000 location wales 1

There's a couple of pieces here that could be pulled out into other PRs. Namely I made a first step at improving the granularity of our sass breakpoints and fixed a semantic issue which are largely tangential to the work here, let me know if you'd prefer that be extracted.

TODO:

- [x] Style programmes list
- [x] Create breadcrumb logic to show active filters, and allow people to return to over10k page
- [x] Update over10k links
- [ ] Update and crop images
- [ ] Translate copy for programmes in CMS based on current funding finder
- [ ] Send UI copy for translation

### CODE REVIEW

File: assets/sass/components/programmes.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        margin-bottom: $spacingUnit / 2;
+
+        .programme-card__media {
+            order: 1;
+            flex: 0 0 100px;
+            margin-right: $spacingUnit;
+        }
+
+        .programme-card__title {
+            order: 2;
```
Comment (davidrapson) [inline assets/sass/components/programmes.scss]: Why do you not like this GitHub?

File: config/locales/cy.json [hunk 2]
```diff
@@ -941,6 +947,13 @@
 					}
 				}
 			}
+		},
+		"programmes": {
+			"title": "Funding Programmes",
```
Comment (davidrapson) [inline config/locales/cy.json]: 👀 Translate me

File: views/layouts/main.njk [hunk 3]
```diff
@@ -11,12 +11,11 @@
     <body{% if bodyClass %} class="{{ bodyClass }}"{% endif %}>
         {% include "../includes/accessibility.njk" %}
 
-        {# main content area #}
-        <main>
+        <div class="global-container">
```
Comment (davidrapson) [inline views/layouts/main.njk]: This is bit of a semantic fix. This arguably shouldn't be a `main` as

1. The _content_ is really the main area, not the whole page
2. We have `role=main` on all container elements inside page templates so semantically we have two mains

**Comment by davidrapson** [comment]
https://github.com/biglotteryfund/blf-alpha/issues/275

File: controllers/funding/programmes.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    };
+}
+
+module.exports = function(config) {
+    return function(req, res) {
+        const lang = req.i18n.__(config.lang);
+
+        const locationParamToTranslation = key => {
+            const regions = {
+                england: req.i18n,
```
Comment (mattandrews) [inline controllers/funding/programmes.js]: Is this right?

File: views/layouts/main.njk (hunk 3 above)
Comment (mattandrews) [reply views/layouts/main.njk]: Fair enough!

File: views/pages/toplevel/over10k.njk [hunk 5]
```diff
@@ -33,19 +33,19 @@
 
             <ul class="unstyled grid grid--padded grid--equal grid--4-up grid--wide-only">
                <li class="grid__item">
-                   <a href="{{ buildUrl('funding/funding-finder?area=England&over=10k') }}"
+                   <a href="{{ buildUrl('funding/programmes?location=england&min=1000') }}"
```
Comment (mattandrews) [inline views/pages/toplevel/over10k.njk]: This was my code originally, but wondering now if we're happy introducing this new URL format here? On the other hand it'd be quite tricky if we kept the same `funding-finder` path as the existing tool, at least until we change it... hmm.

File: assets/sass/globals/typography.scss [hunk 6]
```diff
@@ -10,8 +10,7 @@ h1,
     font-weight: 600;
     @include font-and-leading(18px, 26px);
 
-    // @TODO: Add more granular named breakpoints
-    @media screen and (min-width: 380px) {
+    @media screen and (min-width: xsmall) {
```
Comment (mattandrews) [inline assets/sass/globals/typography.scss]: Is this some CSS magic or is it an error? eg. does it know what `xsmall` means?!

Comment (davidrapson) [reply assets/sass/globals/typography.scss]: Urm… it's sadly an error. A magic error? 💥

File: views/pages/toplevel/over10k.njk (hunk 5 above)
Comment (davidrapson) [reply views/pages/toplevel/over10k.njk]: Yeah, maybe one to discuss. I'm going to roll this change back anyway as I'd like to get this PR out as is, and leaving these URLs in would make it public should we need to deploy…

**Pull request merged**